frontend/src/components/common/Checkbox/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { ChangeEvent } from 'react';
+
+import CheckedIcon from '@/assets/checked.svg';
+import UncheckedIcon from '@/assets/unchecked.svg';
+
+import * as S from './styles';
+
+// NOTE: 공통 컴포넌트에서 이 스타일 속성을 계속 쓰는 것 같은데 이걸 아예 공통 타입으로 빼버릴지 고민
+export interface CheckboxStyleProps {
+  $style?: React.CSSProperties;
+}
+
+interface CheckboxProps extends CheckboxStyleProps {
+  id: string;
+  isChecked: boolean;
+  onChange: (event: ChangeEvent<HTMLInputElement>) => void;
+  name?: string;
+  isDisabled?: boolean;
+}
+
+const Checkbox = ({ id, name, isChecked, isDisabled, onChange, $style }: CheckboxProps) => {
+  return (
+    <S.CheckboxContainer style={$style}>
```

Comment:
```suggestion
    <S.CheckboxContainer $style={$style}>
```
CheckbocContainer의 props타입은 CheckboxStyleProps인데, props를 넣어줄 때는 style에다가 $style를 넣어주고 있어요.

Reply:
인간 디버거 바다 👍👍